go.mod

```diff
@@ -752,3 +752,6 @@ exclude (
 	github.com/knadh/koanf/maps v0.1.1
 	github.com/knadh/koanf/providers/confmap v0.1.0
 )
+
+// Temporarily use a fork of gosnmp for multiple-user traps support
+replace github.com/gosnmp/gosnmp => github.com/zoedt/gosnmp v0.0.0-20231218153121-83a06ce65d5a
```

Comment:
just for clarity: [this is the commit it is pointing to](https://github.com/gosnmp/gosnmp/pull/457/commits/83a06ce65d5af645c209b96f6fa32bd39fbde85d) (most recent on the branch)